swhkd/src/config.rs

```diff
@@ -65,6 +65,8 @@ pub const MODE_STATEMENT: &str = "mode";
 pub const MODE_END_STATEMENT: &str = "endmode";
 pub const MODE_ENTER_STATEMENT: &str = "@enter";
 pub const MODE_ESCAPE_STATEMENT: &str = "@escape";
+pub const MODE_SWALLOW_STATEMENT: &str = "swallow";
+pub const MODE_ONCEOFF_STATEMENT: &str = "onceoff";
```

Comment:
How about `oneoff`? I think it's a bit clearer, but this may just be personal preference. What do you think?

Reply:
Feel free to resolve this directly if you think `onceoff` is better!

Reply:
Sorry I misspelled it 🫣
Already renamed to `oneoff`